ote/src/cljs/ote/views/admin/monitor.cljs

```diff
@@ -1,9 +1,10 @@
-(ns ote.views.monitor
+(ns ote.views.admin.monitor
```

Comment:
Should also controller be renamed to match view, i.e. `ote.app.controller.monitor` to `ote.app.controller.admin.monitor` ?

Reply:
Maybe, but we don't have own .admin. namespace for admin stuff yet. So maybe I do this in some other pr.